### ISSUE TITLE
Authentification

### ISSUE THREAD

**Issue by nilshoerrmann** [pointybeard/api_framework#11]
Are there any plans to allow authentication against either the Symphony users of the [site members](https://github.com/symphonycms/members)?

**Comment by pointybeard** [comment]
Yes, but I haven't put much thought into it just yet. I would imagine a header, something like `Authorization: Token abcd1234`, would be what we're looking at.

I could envisage there being an "auth" endpoint that accepts Symphony credentials in some fashion (shared/secret key, api key, jwt etc) which returns an auth token/session or similar which is used to authenticate subsequent requests.

Not entirely sure yet, and happy for suggestions.
